src/py42/clients/securitydata.py

```diff
@@ -109,7 +109,7 @@ def get_all_plan_security_events(
 
         Returns:
             generator: An object that iterates over tuples whose first element is a :class:`py42.response.Py42Response` object
-            containing a page of events, and whose second element is a cursor.
+            containing a page of events, and whose second element is a string cursor.
```

Comment:
`a string cursor representing the timestamp of the event` or `... with the format of a timestamp` might be good to know, but not a big deal, they will probably figure that out.

Reply:
I think it's a timestamp anyway

Reply:
There's a description of the cursor in the parameter docstring for this function so I think it's probably fine as is.